### ISSUE TITLE
Remmina panel/systray icon renamed

### ISSUE THREAD

**Issue by antenore** [ubuntu/yaru#2940]
I'm the Remmina maintainer, we have renamed the panel (system tray) remmina icon from remmina-panel to remmina-status.

I'm not sure how you could proceed as older versions of Remmina will still have remmina-panel (and remmina-panel-inverted).

Probably adding a new remmina-status.(svg|png) as a copy of the existing remmina-panel.(svg|png) icons?

Sorry for the inconvenience, this change was needed

CC: @ubuntujaggers

**Comment by madsrh** [comment]
If @ubuntujaggers is AFK could you fix this @Jupi007 ? 
Wouldn't the best solution be to rename the old file to and adding a symlink for it?

**Comment by ubuntujaggers** [comment]
Apologies guys, I can do this early next week 🙂

**Comment by ubuntujaggers** [comment]
Oops!  I'll do it now.

**Comment by ubuntujaggers** [comment]
We don't actually seem to have any Remmina icons in our theme since ours was adopted upstream?

**Comment by antenore** [comment]
Yes, that's right @ubuntujaggers but as we have changed the panel icon, it may be not conform to your guidelines and standards?

**Comment by Feichtmeier** [comment]
@antenore hi :wave: is this new version of remminna already in the 21.10 daily repos? So we could see if everything is fine or not

**Comment by antenore** [comment]
Hey @Feichtmeier ! Hope you are doing well.

Yes, it does, we have 1.4.20 thanks to @mfvescovi that is the Remmina downstream maintainer for Debian

**Comment by Feichtmeier** [comment]
@antenore I am fine here hope you too?
Uhh indeed the panel icon now is no longer monochrome but full color and fuzzy at the sides. 
![FLAMESHOT_2021-09-14_15-28](https://user-images.githubusercontent.com/15329494/133266345-b9e3328b-8ca1-40ea-843d-476710b13b43.png)
How does it look in other distributions? Is it correct there?
Otherwise you might fix this upstream instead?
